### PR TITLE
Clean up and refactor code

### PR DESCRIPTION
**Pull request by GuyHadas**
@AviWaiz 
There should be no changes to the actual app here, this is just a giant refactoring of code. Make sure to understand the style guide here (ex: single quotes, whitespace in {}s), let's make sure the code stays consistent throughout.

Please review and leave comments if you have any, and when it's good give a thumbs up and LGTM (Let's get this merged)

### CODE REVIEW

File: frontend/components/track.jsx [hunk 1]
```diff
@@ -5,29 +5,34 @@ class Track extends React.Component{
...
+    return (rank * 55) + 25;
   }
 
   componentDidMount() {
-    this.setState({top: this.props.track.rank * 55 + 25 });
+    this.setState({ top: this.calculateDistanceFromTop(this.props.track.rank) });
   }
 
   componentDidUpdate() {
     var self = this;
```
Comment (AviWaiz) [inline frontend/components/track.jsx]: We should stay consistent with ES6 code every where, so use arrows.

**Comment by AviWaiz** [comment]
@GuyHadas looks good merge

**Comment by GuyHadas** [comment]
@AviWaiz changed to arrow func. Good to merge?

**Review by AviWaiz — APPROVED** [review]

**Comment by AviWaiz** [comment]
👍

**Pull request merged**